packages/data/package.json

```diff
@@ -0,0 +1,54 @@
+{
+	"name": "@automattic/data",
+	"version": "1.0.0",
+	"description": "Data layer for the Automattic ecosystem.",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"types": "dist/types/index.d.ts",
+	"calypso:src": "src/index.ts",
```

Comment:
I think we can get rid of this since there's no `index.ts` file in the `src` folder.